### DISCUSSION TITLE
GUI: Warning double-pass incompatible with PTM scoring, turned off

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#399] [Q&A] [answered]
I get this error message on DIANN1.8.1:
![image](https://user-images.githubusercontent.com/98888664/171147819-b197b3d1-c94f-46ce-94a2-9767782fd64c.png)
Here are more details of the search:
![image](https://user-images.githubusercontent.com/98888664/171147702-37269c6b-235a-4cab-8b04-9e5c6b966bfa.png)

Q1: What does the error mean? Double-pass search is generally incompatible with PTM's ?
Q2: What does 'turned off' mean? Is the double-pass search turned off (so its only single-pass) or are PTMs excluded?
Q3: Where can I find more information on what double-pass exactly does? (Couldn't find a section on the github on it)
Q4: Would the other option (I: PTMs included, II: double pass without PTMs) be potentially better?

**Comment by vdemichev** [comment, marked as answer]
Double pass does not currently work with PTM scoring, so it was turned off in the sense that only a single pass will be used. In general, please use single pass, it's a better tested method.
